runtime/jcl/common/criusupport.c

```diff
@@ -220,6 +238,12 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 		char logFileBuf[STRING_BUFFER_SIZE];
 		char *logFileChars = logFileBuf;
 		IDATA systemReturnCode = 0;
+		jstring checkPointDir = (*env)->GetObjectField(env, criuSupport, vm->criuSupportCheckPointDir);
+		jboolean keepRunning = (*env)->GetBooleanField(env, criuSupport, vm->criuSupportKeepRunning);
+		jboolean shellJob = (*env)->GetBooleanField(env, criuSupport, vm->criuSupportShellJob);
+		jboolean extUnixSupport = (*env)->GetBooleanField(env, criuSupport, vm->criuSupportExtUnixSupport);
+		jint logLevel = (*env)->GetIntField(env, criuSupport, vm->criuSupportLogLevel);
+		jstring logFile = (*env)->GetObjectField(env, criuSupport, vm->criuSupportLogFile);
```

Comment:
add a null check for  `logFile` before `J9_JNI_UNWRAP_REFERENCE(logFile)` and 

```
criu_set_log_level((int)logLevel);
criu_set_log_file(logFileChars);
```

Reply:
For logLevel do you mean check if logLevel >= 0?
Also I can put asserts right? Since the java code should have enforced these

Reply:
nvm